### PR TITLE
Processed output

### PR DESCRIPTION
**Pull request by ocramh**

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -290,6 +314,91 @@ components:
...
+        properties:
+          frame:
+            description: frame number
+            type: integer
+            example: 1
+          timestamp:
+            description: frame timestamp
+            type: string
+            example: '0:00:00.280000'
+          started_at:
```
Comment (raj-shah) [inline docs/openapi.yaml:330]: Not sure if we add this?

File: docs/openapi.yaml [hunk 2]
```diff
@@ -290,6 +314,91 @@ components:
...
+                x2:
+                  description: x axis bounding box bottom right corner
+                  type: integer
+                  example: 40
+                y2:
+                  description: y axis bounding box bottom right corner
+                  type: integer
+                  example: 170
+                label:
+                  description: human readable object label
```
Comment (raj-shah) [inline docs/openapi.yaml:356]: `human-readable`

File: docs/openapi.yaml [hunk 3]
```diff
@@ -290,6 +314,91 @@ components:
...
+                  example: person
+                score:
+                  description: confidence score
+                  type: number
+                  example: 0.9695325493812561
+                id:
+                  description: the object identifier
+                  type: string
+                  example: K_2
+          frame_events:
```
Comment (raj-shah) [inline docs/openapi.yaml:367]: Add activities based on https://github.com/shapes-a1/dvr/pull/138

File: docs/openapi.yaml (hunk 1 above)
Comment (ocramh) [reply docs/openapi.yaml:330]: copy & paste left over

File: docs/openapi.yaml [hunk 4]
```diff
@@ -290,6 +314,91 @@ components:
...
+            type: array
+            items:
+              type: object
+              properties:
+                event_type:
+                  description: type of event detected
+                  type: string
+                  example: violence
+                detected:
+                  description: description of the event
```
Comment (raj-shah) [inline docs/openapi.yaml:377]: functional program description of event

File: docs/openapi.yaml (hunk 3 above)
Comment (raj-shah) [inline docs/openapi.yaml:367]: Add activities based on https://github.com/shapes-a1/dvr/pull/138

File: docs/openapi.yaml (hunk 2 above)
Comment (raj-shah) [inline docs/openapi.yaml:356]: human-readable

File: docs/openapi.yaml (hunk 1 above)
Comment (raj-shah) [inline docs/openapi.yaml:330]: Not sure if we currently add this?

File: docs/openapi.yaml [hunk 5]
```diff
@@ -290,6 +314,91 @@ components:
...
+                  type: string
+                  example: 'TOWARDS ( K_6 , K_2 )'
+                probability:
+                  description: increased or decreased likelihood of the event to happen
```
Comment (raj-shah) [inline docs/openapi.yaml:381]: probability sequence of event

File: docs/openapi.yaml [hunk 6]
```diff
@@ -290,6 +314,91 @@ components:
...
+                  type: string
+                  example: ADD ( 0.8479 )
+                total_probability:
+                  description: overall event confidence score
```
Comment (raj-shah) [inline docs/openapi.yaml:385]: overall event probability/confidence score

File: docs/openapi.yaml [hunk 7]
```diff
@@ -290,6 +314,91 @@ components:
...
+                  type: number
+                  minimum: 0
+                  maximum: 1
+                  example: 0.8479
+                explanation:
+                  description: human readable event explanation
```
Comment (raj-shah) [inline docs/openapi.yaml:391]: human-readable

File: openapi/components/schemas/job/processed_output.yaml [hunk 8]
```diff
@@ -0,0 +1,80 @@
+type: array
```
Comment (raj-shah) [inline openapi/components/schemas/job/processed_output.yaml:1]: same comments as above file

File: docs/openapi.yaml [hunk 9]
```diff
@@ -290,6 +314,95 @@ components:
...
+            example: 1
+          timestamp:
+            description: frame timestamp
+            type: string
+            example: '0:00:00.280000'
+          frame_objects:
+            type: array
+            items:
+              type: object
+              properties:
```
Comment (raj-shah) [inline docs/openapi.yaml:334]: where are label and score?

File: docs/openapi.yaml [hunk 10]
```diff
@@ -290,6 +314,95 @@ components:
...
+                event_type:
+                  description: type of event detected
+                  type: string
+                  example: violence
+                detected:
+                  description: description of the event
```
Comment (raj-shah) [inline docs/openapi.yaml:381]: change description to: functional program description of event

File: docs/openapi.yaml [hunk 11]
```diff
@@ -290,6 +314,95 @@ components:
...
+                  type: string
+                  example: 'TOWARDS ( K_6 , K_2 )'
+                probability:
+                  description: increased or decreased likelihood of the event to happen
```
Comment (raj-shah) [inline docs/openapi.yaml:385]: change description to: probability sequence of event

File: docs/openapi.yaml [hunk 12]
```diff
@@ -290,6 +314,95 @@ components:
...
+                  type: string
+                  example: ADD ( 0.8479 )
+                total_probability:
+                  description: overall event confidence score
```
Comment (raj-shah) [inline docs/openapi.yaml:389]: change description to: overall event probability/confidence score

File: openapi/components/schemas/job/processed_output.yaml (hunk 8 above)
Comment (raj-shah) [reply openapi/components/schemas/job/processed_output.yaml:1]: Same additional comments as above. Also missing label and score in frame_objects

File: docs/openapi.yaml [hunk 13]
```diff
@@ -290,6 +314,95 @@ components:
...
+                activities:
+                  type: array
+                  description: >-
+                    collection of the top 5 detected activities in the current
+                    frame
+                  items:
+                    type: object
+                    properties:
+                      label:
+                        description: human-readable object label
```
Comment (raj-shah) [inline docs/openapi.yaml:360]: human-readable activity label

**Pull request merged**